vulture_whitelist.py

```diff
@@ -0,0 +1,21 @@
+from jefferson.core import (
+    JFFS2_COMPAT_MASK,
+    JFFS2_COMPR_COPY,
+    JFFS2_COMPR_DYNRUBIN,
+    JFFS2_COMPR_RUBINMIPS,
+    JFFS2_FEATURE_ROCOMPAT,
+    JFFS2_FEATURE_RWCOMPAT_COPY,
+    data_crc_match,
+    name_crc_match,
+    node_crc_match,
+)
+
+data_crc_match
```

Comment:
I don't see these used, shouldn't these be removed from the whitelist?
We can merge with this, but maybe with a TODO comment or new issue to fix it next PR.

Reply:
This can be marked as resolved for this PR, can be improved next.

Reply:
I added a TODO comment and will create an issue for it. I did not want to touch the logic within this PR, even if it's to remove dead code.